### PR TITLE
sort category

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Sort Category
## Reference Requirements
- Task ID: #87

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
```
explain select * from `categories` order by `categories`.`name` asc limit 15 offset 0\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: categories
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 30
     filtered: 100.00
        Extra: Using filesort
1 row in set, 1 warning (0.08 sec)
```
- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: tests/Browser/Pages/Backend/UpdateCategoryTest.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ public function testValidateForInput()
                 ->click('.button-edit-category')
                 ->type('categoryName', '');
             $browser->press('Update')
-               ->pause(500)
+               ->pause(1000)
```
Comment (at-vietpham) [inline tests/Browser/Pages/Backend/UpdateCategoryTest.php]: Không cân dùng `pause`

File: resources/views/admin/categories/index.blade.php [hunk 2]
```diff
@@ -41,7 +41,7 @@
                     </tbody>
                 </table>
                 <div class="text-center">
-                    {{ $categories->links() }}
+                    {{ $categories->appends(\Request::except('page'))->render() }}
```
Comment (at-vietpham) [inline resources/views/admin/categories/index.blade.php]: Em có thể `appends` ngay ở Controller.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-vietpham** [comment]
LGTM 🔢

**Pull request merged**